### ISSUE TITLE
(${val}>=err()) gives parse exception

### ISSUE THREAD

**Issue by at055612** [gchq/stroom-expression#20]
```
java.text.ParseException: Unexpected token type 'GREATER_THAN_OR_EQUAL_TO'
	at stroom.dashboard.expression.v1.ParamFactory.create(ParamFactory.java:41)
	at stroom.dashboard.expression.v1.ExpressionParser.getParam(ExpressionParser.java:231)
	at stroom.dashboard.expression.v1.ExpressionParser.applyEquality(ExpressionParser.java:335)
	at stroom.dashboard.expression.v1.ExpressionParser.getParam(ExpressionParser.java:247)
	at stroom.dashboard.expression.v1.ExpressionParser.applyEquality(ExpressionParser.java:333)
	at stroom.dashboard.expression.v1.ExpressionParser.getParam(ExpressionParser.java:247)
	at stroom.dashboard.expression.v1.ExpressionParser.getFunction(ExpressionParser.java:179)
	at stroom.dashboard.expression.v1.ExpressionParser.processObjects(ExpressionParser.java:109)
	at stroom.dashboard.expression.v1.ExpressionParser.parse(ExpressionParser.java:76)
```

**Comment by at055612** [comment]
Similar exception for `<=`.  `<` and `>` work ok though.

**Comment by at055612** [comment]
fixed in > `1.3.2`
